app/src/main/java/com/android/feedme/model/data/RecipeRepository.kt

```diff
@@ -66,14 +67,15 @@ class RecipeRepository(private val db: FirebaseFirestore) {
   }
 
   /**
-   * Fetch all the recipes of the given List of Ids
+   * Fetches the top rated recipes from Firestore. The recipes are ordered by their rating in
```

Comment:
This method would be for saved recipes instead of top rated ones

Reply:
Nice catch, I'll fix it.